ui-components/src/tables/TableCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { Heading } from "../headers/Heading"
+import { Button } from "../actions/Button"
+import { t } from "../helpers/translator"
+import "./TableCard.scss"
+
+export interface TableCardProps {
+  title: string
+  children?: React.ReactNode
+  elementsQty: number // to show/hide an empty state
```

Comment:
This prop is really tied to our use for this component - that it will add / remove items, but visually this component could be used for other things. Could we toggle that `add items` state on/off through the children? Open to other suggestions!

Reply:
To make this component more reusable, I can get rid of `Add item` button and just create a prop like `footer: React.ReactNode`. You can pass everything in the parent component (e.g button), but the top padding still is controlled by this card component. What do you think?

Reply:
That sounds great!